### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by lourdesVillca**

### CODE REVIEW

**Comment by lourdesVillca** [comment]
@carledriss ready to code review

File: src/main/java/org.fundacionjala.katapractice/HighAndLow.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    }
+
+    /**
+     * Method to add get the high and low number.
+     *
+     * @param values the String that contain the values.
+     * @return the result of the calculation that is the high and low number.
+     */
+    public static String getHighLow(final String values) {
```
Comment (carledriss) [inline src/main/java/org.fundacionjala.katapractice/HighAndLow.java:23]: Could you try the following code?

``` java
        IntStream intStream = Arrays.stream(values.split(" ")).mapToInt(Integer::parseInt);
        int high = intStream.max().getAsInt();
        int low = intStream.min().getAsInt();
        return String.format("%d %d", high, low);
```

File: src/main/java/org.fundacionjala.katapractice/DnaStrand.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    }
+
+    /**
+     * This method to make the complements given a dna.
+     *
+     * @param dna the dna to be complement.
+     * @return the complemented dna.
+     */
+    public static String makeComplement(final String dna) {
+        final String result = dna.replaceAll("A", "t").replaceAll("C", "g").replaceAll("T", "a").replaceAll("G", "c");
```
Comment (carledriss) [inline src/main/java/org.fundacionjala.katapractice/DnaStrand.java]: Could you try the following code?

``` java
        return dna.replaceAll("A", "t")
                .replaceAll("C", "g")
                .replaceAll("T", "a")
                .replaceAll("G", "c")
                .toUpperCase();
```

**Comment by carledriss** [comment]
@lourdesVillca 
After fixing minor observations.
Looks good to me!

File: src/main/java/org.fundacionjala.katapractice/HighAndLow.java (hunk 1 above)
Comment (lourdesVillca) [reply src/main/java/org.fundacionjala.katapractice/HighAndLow.java:23]: I have tried  to do this change but there is a problem when I compile, the following exception is displayed "java.lang.IllegalStateException: stream has already been operated upon or closed" 
I research about the problem and, I read that a stream is possible to use only one time.

**Comment by lourdesVillca** [comment]
@carledriss ready to code review,  I had problems to fix your comments in HighAndLow.java classs

File: src/main/java/org.fundacionjala.katapractice/HighAndLow.java (hunk 1 above)
Comment (carledriss) [reply src/main/java/org.fundacionjala.katapractice/HighAndLow.java:23]: @lourdesVillca 
Alright.
I didn't know that. thank you for letting me know. :)

**Pull request merged**